lib/wikidata_lookup.rb

```diff
@@ -148,12 +148,12 @@ def other_fields_for(result)
   end
 
   private
-  WDQ_URL = 'https://wdq.wmflabs.org/api'
+  WDQ_URL = 'https://query.wikidata.org/sparql'
 
   def wdq(query)
```

Comment:
Done in https://github.com/everypolitician/everypolitician-data/commit/fc614096d84ef5d3186099289c8dc3c42de33689